assets/js/script.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable max-classes-per-file */
```

Comment:
-  Please don't disable the linter errors, fix them manually instead. It will help you to write clean code. 👍🏽
- Kindly remove all unnecessary comments to keep your code clean 👍🏽

Reply:
well fixed.
thanks for feedback dear @rotimiazeez